src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -48,6 +50,9 @@ public static class KnownRoutes
     public const string SeriesMetadataRoute = SeriesRoute + "/" + MetadataSegment;
     public const string InstanceMetadataRoute = InstanceRoute + "/" + MetadataSegment;
 
+    public const string InstanceRenderedRoute = InstanceRoute + "/" + RenderedSegment;
+    public const string FrameRenderedRoute = InstanceRoute + "/frame/" + SingleFrameIdRouteSegment + "/" + RenderedSegment;
```

Comment:
This should be `frames` right?

Reply:
It was initially frames but as we discussed offline we wanted it to be frame since we only support 1 frame

Reply:
updated